backend/src/controllers/rides.ts

```diff
@@ -93,6 +105,18 @@ export const createRide = async (req: CustomRequest, res: Response): Promise<voi
   ride.rideId = rideId;
   ride.requestTimeStamp = new Date();
 
+  if (ride.pickupDateTime) {
+    ride.pickupDateTime = new Date(ride.pickupDateTime);
+  } else {
+    ride.pickupDateTime = new Date(fixTimeUpDayjs());
+  }
+
+  if (ride.relevantTime) {
```

Comment:
@aviv-taori No need to set the value from the body again, and I'm not sure `body.ride` exists.